src/pages/Account/Profile.tsx

```diff
@@ -115,11 +114,11 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
   return (
     <div className="Profile">
       <h2 className="login-welcome-title">
-        Hey, {githubHandle}. We’re so glad you’re here! Upload or take a photo
-        and share your Summit snap here.
+        Hey, {githubHandle}. We’re so glad you’re here!
       </h2>
       <div className="login-welcome-subtitle">
-        {`Swing back and edit your profile anytime you like.`}
+        We&apos;ve pre-populated your profile with details from Okta - please{" "}
+        <br /> review and click Next if it all looks good.
```

Comment:
`<br />` Shouldn't be used here like this.. If you want the text to 'flow' to a new line, then the 'container' it is in shouldn't be as wide.

Reply:
@mobilevinay suggested this so I went with it. I could make a new PR for this, but I think the registration page is already done being used (as in everyone has already registered for the event so they won't go to this page, but I might be wrong)